### ISSUE TITLE
Image Link when posting to social media not present

### ISSUE THREAD

**Issue by zachbogart** [jtr13/EDAV#128]
I'm finding it difficult to make the image appear when posting to social accounts or having the cover image appear on the bookdown page.

Looking at the page source, the html suggests that it has access (a twitter image field is populated, for example). I'm going to try to update the url path and see if that helps.

**Comment by jtr13** [comment]
Not critical... we can keep it on the back burner.

**Comment by zachbogart** [comment]
Checked your twitter post and bookdown archive and it is live now. Just took some time to get out there.

https://twitter.com/jtrnyc/status/1052355433422344192

**Comment by jtr13** [comment]
Looks great!! How did you do it?

**Comment by zachbogart** [comment]
I think since we are using a custom domain, it was having trouble. In index.Rmd, I just said the url was "edav.info/" but it wanted a full url like 'https://edav.info/'. This automatically joins with the cover image specified in "cover-image". Also, it took some time for the bookdown archive and twitter to notice and take effect.

**Comment by jtr13** [comment]
Yeah, that's the frustrating part: wait an unspecified amount of time to see if your fix worked!   Fwiw, I'm making progress on figuring out (one of) the cause(s) of my merge conflicts. I realized that when I make changes on GitHub through a patch, as I just did today, even when I do `git pull`, it doesn't update the Joyce branch locally, only master. To update Joyce I have to `git merge master` and then `git push` to push the local updates from master to Joyce back to the Joyce branch on GitHub:

Patch on GitHub --> master --> pull changes to local --> merge to branch --> push to branch on GitHub
